src/node/rpc/endpoint_registry.h

```diff
@@ -604,23 +604,6 @@ namespace ccf
      */
     void install(Endpoint& endpoint)
     {
-      if (endpoint.authn_policies.empty())
-      {
-        LOG_FAIL_FMT(
-          "Endpoint {} /{} does not have any authentication policy",
-          endpoint.dispatch.verb.c_str(),
-          endpoint.dispatch.uri_path);
-      }
-
-      // A single empty auth policy is semantically equivalent to no policy, but
-      // no policy is faster
-      if (
-        endpoint.authn_policies.size() == 1 &&
-        endpoint.authn_policies.back() == empty_auth_policy)
-      {
-        endpoint.authn_policies.pop_back();
-      }
```

Comment:
I think this section should be kept?

Reply:
I thought empty_auth was just an empty vector now?

Reply:
Ah, no!

Reply:
`no_auth_required` _is_ an empty vector, but if we end up with the vector `{empty_auth_policy}` through some other way, we can optimise it here?